_includes/v20.2/sql/unsupported-postgres-features.md

```diff
@@ -1,9 +1,11 @@
 - Stored procedures and functions
 - Triggers
 - Events
-- User-defined functions
+- User-defined functions (UDF)
 - FULLTEXT functions and indexes
-- Drop primary key
+- Drop primary key*
```

Comment:
I think we usually do superscript numbered notes for this situation, but I can't find it documented anywhere. Might be worth a post in #docs-style.

Reply:
Followed Rich's advice [here](https://cockroachlabs.slack.com/archives/C01NFCXULTX/p1648146216252599).